gateway/ln-gateway/Cargo.toml

```diff
@@ -54,6 +54,7 @@ fedimint-mint-client = { path = "../../modules/fedimint-mint-client" }
 fedimint-dummy-client = { path = "../../modules/fedimint-dummy-client" }
 fedimint-wallet-client = { path = "../../modules/fedimint-wallet-client" }
 futures = "0.3.24"
+lazy_static = "1.4.0"
```

Comment:
Not enthusiastic about it.

Reply:
We already depend on it transitively in a bunch of places and it's a well-known crate, so not too big of a deal imo.

Reply:
Removed